src/pint/models/timing_model.py

```diff
@@ -406,12 +415,21 @@ def free_params(self, params):
 
     def match_param_aliases(self, alias):
         """Return the parameter corresponding to this alias."""
-        for p in self.params:
+        # Search the top level first.
+        for p in self.top_level_params:
             if p == alias:
                 return p
             if alias in getattr(self, p).aliases:
                 return p
-        raise ValueError("{} is not recognized as a parameter or alias".format(alias))
+        # if not in the top level, check parameters.
+        for cp in self.components.values():
+            pint_par = cp.match_param_aliases(alias)
```

Comment:
Can `cp.match_param_aliases` be modified to raise an exception if there is no match?